openlane/steps/openroad.py

```diff
@@ -1806,10 +1806,15 @@ class CTS(ResizerStep):
             ),
             Variable(
                 "CTS_MAX_CAP",
-                Decimal,
-                "Defines the maximum capacitance, used in CTS.",
+                Optional[Decimal],
+                "Overrides the maximum capacitance used in CTS.",
```

Comment:
If it overrides, where it the default value coming from?

Reply:
Added to description